tests/shared.py

```diff
@@ -186,10 +186,14 @@ def run(self, *command):
 
 
 class GitRepo(Repo):
-    def __init__(self, content_dir):
+    def __init__(self, content_dir, **init_default_branch):
```

Comment:
The `**` syntax tells Python to collect _any_ keyword arguments into a dictionary called `init_default_branch`. What we'd prefer is to define a single default argument, like this:

```python
def __init__(self, content_dir, init_default_branch=None):
```

Then we can test it like this:

```python
if init_default_branch is not None:
```

There might also be some more specialized syntax we can take advantage of here, see below...

Reply:
Nice, one new thing learned.